addons/60fps/addon.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "60FPS",
```

Comment:
Maybe it could be a more descriptive name?

Reply:
how about 60fps player mode?